packages/server/graphql/private/mutations/connectSocket.ts

```diff
@@ -0,0 +1,87 @@
+import {InvoiceItemType, SubscriptionChannel} from 'parabol-client/types/constEnums'
+import adjustUserCount from '../../../billing/helpers/adjustUserCount'
```

Comment:
Should we consider using a typescript path to make these `../../../` imports simpler to read?